packages/common/infra/src/modules/feature-flag/constant.ts

```diff
@@ -153,6 +154,14 @@ export const AFFINE_FLAGS = {
     configurable: true,
     defaultState: false,
   },
+  enable_mobile_keyboard_toolbar: {
+    category: 'blocksuite',
+    bsFlag: 'enable_mobile_keyboard_toolbar',
+    displayName: 'Mobile Keyboard Toolbar',
+    description: 'Enables the mobile keyboard toolbar.',
+    configurable: false,
+    defaultState: isMobile,
```

Comment:
I think we should add a filter method for the flag, like this one that only needs to be available on mobile. @EYHN